rtb/piano-di-progetto/src/pianificazione.tex

```diff
@@ -1,7 +1,578 @@
+    row{odd}={bg=white},
+    row{even}={bg=lightgray},
+    row{1}={bg=black,fg=white},
+    row{8}={bg=black,fg=white}
+    }
+    
+    Nominativo    & Re & Am & An & Pg & Pr & Vf & Ore Totali \\ \hline
+    Alberto C.    & -  & -  & 2  & -  & -  & 3  & 5 \\ \hline
+    Bibal El M.   & 2  & 2  & 6  & -  & -  & 5  & 15 \\ \hline
+    Alberto M.    & -  & -  & 4  & -  & -  & 5  & 9 \\ \hline
+    Alex S.       & 5  & 5  & -  & -  & -  & -  & 10 \\ \hline
+    Iulius S.     & -  & -  & 4  & -  & -  & 5  & 9  \\ \hline
+    Giovanni Z.   & -  & -  & 5  & -  & -  & 5  & 10 \\ \hline
+    Totale        & 7  & 7  & 21 & 0  & 0  & 23 & 58 \\ \hline
+
+\end{tblr}
+
+\textbf{Consuntivo economico}
+
+\begin{tblr}{
+colspec={|X[5cm]|X[3.5cm]|X[1.5cm]|X[3.5cm]},
+row{odd}={bg=white},
+row{even}={bg=lightgray},
+row{1}={bg=black,fg=white},
+row{8}={bg=black,fg=white}
+}
+
+Ruolo & Costo orario (€/h) & N. Ore & Costo totale (€)  \\ \hline
+Responsabile      & 30,00 &  7  &   210,00 \\ \hline
+Amministratore    & 20,00 &  7  &   140,00 \\ \hline
+Analista          & 25,00 &  21 &   525,00 \\ \hline
+Progettista       & 25,00 &  0  &     0,00 \\ \hline
+Programmatore     & 15,00 &   0 &     0,00 \\ \hline
+Verificatore      & 15,00 &  23 &   345,00 \\ \hline
+Totale & \SetCell[c=1]{c} & 192 & 1.220,00 \\ \hline
+
+\end{tblr}
+
+\textbf{Gestione dei ruoli}
+
+\paragraph{}
+Come si può vedere dalla suddetta tabella,in questo quarto periodo il gruppo ha concentrato buona parte del monte-ore nello svolgimento
+del ruolo di Analista e Verificatore: per quanto concerne il primo, i membri hanno speso una parte consistente delle proprie
+risorse  (36\%) al fine di completare la stesura di tutta la documentazione e di effettuare le correzioni all'Analisi dei Requisiti
+in seguito alla revisione del prof. Cardin; di conseguenza, una parte cospicua di risorse (39\%) è stata utilizzata per la fase di verifica 
+e validazione della documentazione sopracitata; il gruppo ha inoltre riservato il 12\% delle ore al ruolo del Responsabile, data l'importanza di un efficace
+ed efficiente orchestrazione delle attività del gruppo in vista della seconda parte della RTB, ed una quantità uguale di ore alla figura dell'Amministratore, al fine
+di evitare potenziali conflitti negli items di versionamento prodotti dai differenti membri.
+
+\paragraph{Gestione dei rischi}
 
+\begin{itemize}
+    \item \textbf{Rischio verificatosi:} Conflitti decisionali.
+    \begin{itemize}
+        \item \textbf{Esito PdC:}In seguito alla revisione della Technology Baseline effettuate dal prof. Cardin, sono emerse criticità nella progettazione
+        del PoC e in particolare nella scelta dello stack tecnologico; è stata quindi avviata una discussione tra i membri del gruppo su come risolvere tali criticità,
+        consci della tabella di marcia da seguire rigorosamente al fine di completare la revisione PB evitando ulteriore ritardo a quello già accumulato. Data la 
```

Comment:
```suggestion
        consci della tabella di marcia da seguire rigorosamente al fine di completare la revisione RTB evitando ulteriore ritardo a quello già accumulato. Data la 
```

Intendevi RTB?

Reply:
in realtà intendevo proprio la PB ,dato che abbiamo la data del 30 aprile, non so se avessimo mai messo giù una data ufficiale dell'RTB